modules/sov-ibc/src/rpc/context.rs

```diff
@@ -22,9 +23,93 @@ impl<'a, S> ProvableContext for IbcContext<'a, S>
 where
     S: Spec,
 {
-    /// TODO: Should figure out how can access the proof from the context
-    fn get_proof(&self, _height: Height, _path: &Path) -> Option<Vec<u8>> {
-        Some(vec![])
+    fn get_proof(&self, height: Height, path: &Path) -> Option<Vec<u8>> {
```

Comment:
Just a heads-up, using `get_proof` in the Sovereign SDK case isn't the efficient approach. It requires us to make two storage calls for an RPC response — once for the value and then for the proof. In fact, ibc-rs `QueryContext` isn't well-suited for this case. We should consider implementing the obtaining of proof for each endpoint within their respective methods.

Reply:
ah. I just realized, the issue talks about bypassing `QueryContext` completely.

Reply:
I think this will duplicate efforts if I do this directly on `sovereign-ibc` - the changes needed for `QueryContext` will be very similar.

Reply:
Yep, either bypassing the `QueryContext` or refactoring it at ibc-rs (if it's not gonna be much disruptive IMO) to support this case.